CoreUtilities/src/au/gov/asd/tac/constellation/utilities/icon/ConstellationIcon.java

```diff
@@ -415,6 +481,50 @@ private ThreeTuple<Integer, Integer, Color> buildCacheKey(final int size, final
     private byte[] retrieveIconData(final IconData iconData, final int size, final Color color) {
         return iconData.getData(size, color);
     }
+    
+    /**
+     * Applies a color filter to enhance colors in a raster image. 
+     * This effect is achieved in open GL within the graph view, however is replicated here
+     * to achieve similar effects within other non-GL plugins such as the SVGExportPlugin.
+     * 
+     * @param original
+     * @param color
+     * @return 
+     */
+    private byte[] applyColorFilter(final byte[] original, final Color color) {
+        final ByteArrayInputStream bais = new ByteArrayInputStream(original);
+        try {
+            final BufferedImage img = ImageIO.read(bais);
+            if (img == null || color == null) {
+                return original;
+            } else {
+                final BufferedImage coloredImage = new BufferedImage(img.getWidth(), img.getHeight(), BufferedImage.TYPE_4BYTE_ABGR);
+                for (int x = 0; x < img.getWidth(); x++) {
+                    for (int y = 0; y < img.getHeight(); y++) {
+                        final Color pixel = new Color(img.getRGB(x, y), true);
+                        final float redFilter = (color.getRed() / 255F);
+                        final float blueFilter = (color.getBlue() / 255F);
+                        final float greenFilter = (color.getGreen() / 255F);
+
+                        final float blendRed = pixel.getRed() * redFilter / 255;
+                        final float blendGreen = pixel.getGreen() * greenFilter / 255;
+                        final float blendBlue = pixel.getBlue() * blueFilter / 255;
+                        final Color blend = new Color(blendRed, blendGreen, blendBlue, pixel.getAlpha() / 255.0F);
+                        coloredImage.setRGB(x, y, blend.getRGB());
+                    }
+                }
+                final ByteArrayOutputStream os = new ByteArrayOutputStream();
+                try {
+                    ImageIO.write(coloredImage, ConstellationIcon.DEFAULT_ICON_FORMAT, os);
+                } catch (final IOException ex) {
+                    return original;
+                }
```

Comment:
given the catch in the outer try is exactly the same, you probably can omit the inner try (if `ImageIO.write` throws an IOException then it will be caught in the outer try)